backend/lib/spree/backend/tabs/product/prices.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require 'spree/backend/tabs/product/base'
+
+module Spree
+  module Backend
+    module Tabs
+      class Product
+        class Prices < Base
+          def name
+            'Prices'
+          end
+
+          def presentation
+            view_context.plural_resource_name(Spree::Price)
```

Comment:
@tvdeyen unfortunately, we don't have this `plural_resource_name` method into `I18n`, it's something we've built as a helper [here](https://github.com/solidusio/solidus/blob/1ec3fac0b65dbbf97be3dbf119b0f0133017484a/core/app/helpers/spree/base_helper.rb#L150) and I can't think of a clean way to use it here without using the `view_context`, do you have any suggestion?

Reply:
Yes, you should be able to do

```rb
Spree::Price.model_name.human(count: Spree::I18N_GENERIC_PLURAL)
```

no?

Reply:
Sure, but there's this not ideal duplication of logic which I'd like to avoid.

Reply:
Not all duplication is bad. I consider this a good thing actually. We do not want to rely on that the `view_context` has magically access to the `plural_resource_name` helper. I kind of embrace explicitness over Rails magic lately ;)